### ISSUE TITLE
Question: is there a way to reduce the distance when using fade animation direction up?

### ISSUE THREAD

**Issue by josuearrieta** [morellodev/react-awesome-reveal#183]
The distance is a bit too much, is it a way to reduce it a little bit?

**Comment by SteveAtKlover** [comment]
i agree this is needed. i tried `react-reveal` but it hasn't been updated in 5 years and doesn't support react 17 and up, so im using this one, but it doesn't have a distance prop.
